geopyspark-backend/project/plugins.sbt

```diff
@@ -4,7 +4,6 @@ resolvers ++= Seq(
 )
 
 addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.0")
-
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.5")
-
 addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.2")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.1.0-M10")
```

Comment:
Used coursier to speed up deps resolution, as there was a chance that the build could be time outed